CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# Welcome to CEDAR Contributions Page 
+Thank you for your interest in contributing to CEDAR. 
+
+Below you will find instructions and guidelines we encourage you to follow. 
+Prior to contribute to the project, please let us know what you are interested in working on, by either:
+1. Posting your intented feature in an [issue](https://github.com/microsoft/CEDAR.GitHub.Collector/issues/new) 
+- We will discuss the feature (design, architecture etc.) and answer any of your questions. 
+- After a mutual agreement on the plan, start coding. 
+- Our team will move the issue to the [Current quarter bucket](https://github.com/microsoft/CEDAR.GitHub.Collector/projects/1) 
+
+2. Find a [existing issue](https://github.com/microsoft/CEDAR.GitHub.Collector/issues) you would like to work on. 
+- Chose the issue and comment below you would like to work on it 
+- Please let us know if you have any questions or would like more context 
+- After this our team will move the issue to the [Current quarter bucket](https://github.com/microsoft/CEDAR.GitHub.Collector/projects/1) 
+
+*Note: To setup your CEDAR repository please look at [readme.md](https://github.com/microsoft/CEDAR.GitHub.Collector/blob/main/README.md)* 
+
+# Creating a PR 
+*Note: you can commit them to your remote branch and request that your changes be merged into the CEDAR.GitHub.Collector repository.*
```

Comment:
Only team members can commit/push to remote branches. External people must do this through forks. It would be nice if you can clarify this.